app/models/post.rb

```diff
@@ -1,17 +1,19 @@
 class Post < ApplicationRecord
-  belongs_to :user, foreign_key: 'user_id'
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
   has_many :comments
   has_many :likes
 
-  after_save :increment_posts_counter
+  validates :title, presence: true, length: { maximum: 250 }
 
-  def recent_comments(limit = 5)
-    comments.order(created_at: :desc).limit(limit)
+  def recent_comments
```

Comment:
- [ ] As per these [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=exceed%20250%20characters.-,CommentsCounter%20must%20be%20an%20integer%20greater%20than%20or%20equal%20to%20zero,LikesCounter%20must%20be%20an%20integer%20greater%20than%20or%20equal%20to%20zero.,-Add%20unit%20specs), please add validations here in the post model:
    - CommentsCounter must be an integer greater than or equal to zero.
    - LikesCounter must be an integer greater than or equal to zero.

Reply:
Done